pvlib/iam.py

```diff
@@ -91,21 +91,22 @@ def ashrae(aoi, b=0.05):
     return iam
 
 
-def physical(aoi, n=1.526, K=4., L=0.002):
+def physical(aoi, n=1.526, K=4.0, L=0.002, *, n_ar=None):
```

Comment:
The ‘single star’ syntax indicates the end of positional parameters and enforces the user to use a keyword for all following parameters, see https://peps.python.org/pep-3102.
In other words `physical(aoi, 1.526, 4.0, 0.0032, 1.29)` is not allowed and will fail with a `TypeError`. One needs to explicitly add the `n_ar` keyword, like `physical(aoi, 1.526, 4.0, 0.0032, n_ar=1.29)`.

I think of using `*` to be good practice for functions with many parameters, so I typically do it when adding new parameters, but it may be overkill. On the other hand, removing the asterisk later is easy and doesn't break anything, but adding it later would be a breaking change.

Reply:
Huh, interesting, I certainly didn't know that. I'm all for keeping it. Thanks for the explanation.

Reply:
I think it will be a first for pvlib, but I'm okay with keeping it.  If anyone complains we can always revert it as @kdebrab points out.